### PR TITLE
Automatically set webhook

### PR DESCRIPTION
**Pull request by kemskems**
This PR lets bot to automatically set its webhook URL if not in development.

### CODE REVIEW

File: tululbot/__init__.py [hunk 1]
```diff
@@ -35,3 +36,7 @@ def handle_uncaught_exception(error):
         bot.send_message(chat_id, traceback.format_exc())
 
     return 'OK'
+
+
+if app.config['APP_ENV'] != 'development':
+    bot.set_webhook(webhook_url_base+webhook_url_path)
```
Comment (fushar) [inline tululbot/__init__.py]: - [ ] spaces around operator?

Comment (fushar) [inline tululbot/__init__.py]: - [ ] or `'{}{}'.format(...)`

**Comment by fushar** [comment]
🚢

**Comment by kemskems** [comment]
Fixed. Please review @fushar.

**Pull request merged**